docs/source/apps.rst

```diff
@@ -0,0 +1,14 @@
+******
+Models
+******
+
+.. currentmodule:: seirmo
+
+The apps class create an app to visualise the model and its simulation.
+
+Overview:
+
+- :class:`_SliderComponent`
```

Comment:
Can you delete this file for now? It doesn't really make sense to register a private class in the docs. (or make change the name of the class from `_SliderComponent` to `SliderComponent`